docker-compose.yml

```diff
@@ -1,8 +1,8 @@
 version: '3'
 services:
-  emu:
+  raven:
     build: .
-    image: huard/raven
+    image: pavics/raven
```

Comment:
I would rather call this image `local_raven`.  Doing it the current way will overlay our local build with the real `pavics/raven:latest` from official Docker Hub build.